ci/utils.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+is_pod_exist() {
+    local POD_NAME=$1
+    kubectl get pods -A 2>&1 | grep $POD_NAME
+    return $?
+}
+
+is_runtimeclass_exist() {
+    local RUNTIMECLASS_NAME=$1
+    kubectl get runtimeclass 2>&1 | grep $RUNTIMECLASS_NAME
+    returnh $?
+}
+
+wait_pod_ready() {
+    local TIMEOUT_SECS=$1
+    local POD_NAME=$2
+
+    timeout $TIMEOUT_SECS bash -c '
+        pod_name=$0
+        while [ true ]
+        do
+            pod_info=$(kubectl get pods -A 2>&1 | grep $pod_name | awk '"'"'{print $3" "tolower($4)}'"'"')
```

Comment:
Perhaps we can check whether the pod is ready by checking whether the pod is running and in the ready state
for example:
```
          while true; do
            count=$(kubectl get pod $pod_name| grep "Running" | grep -c "1/1")
            if [ $count -eq 1 ]; then
              break
            fi
            sleep 5
          done
```

Reply:
I checked the status of the specific pod in the `awk` command, `$3` meaning n/N , and `$4` meaning containercreating/running etc. For coco operator, the ready pod number should be 2/2. And for cc runtimeclass, the ready pod number should be 1/1. For the sake of generality, I implemented it by getting the ready pod numbers automatically.